qiskit/utils/optionals.py

```diff
@@ -292,3 +292,9 @@ def _nlopt_callback(available):
     ("pdftocairo", "-v"),
     msg="This is part of the 'poppler' set of PDF utilities",
 )
+
+HAS_CONSTRAINT = _LazyImportTester(
+    "constraint",
+    name="python-constraint",
+    install="pip install python-constraint",
+)
```

Comment:
Let's add a line of documentation in the module-level docstring as well for the new object.

If you're bothered about such things, the definitions of all the `HAS_*` variables are split into the imports and the subprocesses, and within those they're in alphabetical order, but trying to maintain that long-term might be a lost cause.  The documentation we should keep alphabetical, though.

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/7733/commits/5f44ae870f8ba7ba77e8e5f9ad954a913696f61d